package/crds/postgresql.appcat.vshn.io_postgresqlstandaloneconfigs.yaml

```diff
@@ -0,0 +1,129 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
+  name: postgresqlstandaloneconfigs.postgresql.appcat.vshn.io
+spec:
+  group: postgresql.appcat.vshn.io
+  names:
+    kind: PostgresqlStandaloneConfig
+    listKind: PostgresqlStandaloneConfigList
+    plural: postgresqlstandaloneconfigs
+    singular: postgresqlstandaloneconfig
+  scope: Cluster
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: A PostgresqlStandaloneConfig configures a PostgresqlStandalone
+          provider.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: A PostgresqlStandaloneConfigSpec defines the desired state
+              of a PostgresqlStandaloneConfig.
+            properties:
+              defaultChart:
```

Comment:
Why I don't see these fields in the sample?

Reply:
The sample doesn't define all fields yet. Also I'm not yet sure about this field anyway, I might come up with something different, that's why I didn't include it yet.

Reply:
The sample contains now more fields.